simpeg_drivers/driver.py

```diff
@@ -612,7 +612,16 @@ def configure_dask(self):
             dconf.set(scheduler="threads", pool=ThreadPool(n_cpu))
 
     @classmethod
-    def start(cls, filepath: str | Path | InputFile):
+    def start(cls, filepath: str | Path | InputFile, run=True) -> InversionDriver:
```

Comment:
I think we should make a build method that just returns the driver, and then a start that build the driver and runs it.  It saves the logic here and reduces code duplication when we override the base drivers with the ability to switch applications based on data in input_file (which we should try to generalize!).

Reply:
roger